shell/lib/db-deprecated.js

```diff
@@ -74,6 +74,7 @@ if (Meteor.isServer) {
 if (Meteor.isClient) {
   Session.setDefault("shrink-navbar", false);
   globalGrains = new ReactiveVar([]);
+  Session.set("shrink-navbar", window.localStorage.getItem("shrink-navbar") === "true");
```

Comment:
Will this be reactive? It seems like it wouldn't be, which would mean that different browser tabs would get out of sync on this, which is a little sad, but maybe that's life.